yt/frontends/tipsy/io.py

```diff
@@ -209,7 +212,7 @@ def _read_particle_data_file(self, data_file, ptf, selector=None):
 
         # close all file handles
         f.close()
-        for fh in list(aux_fh.values()):
+        for fh in list(_aux_fh.values()):
```

Comment:
since you touched this line, might as well get rid of the unneeded cast to `list`
```suggestion
        for fh in _aux_fh.values():
```